### ISSUE TITLE
No icons after upgrade to chronograf 1.7.2

### ISSUE THREAD

**Issue by youmil** [influxdata/chronograf#4782]
Hi,
I just upgrade to chronograf 1.7.2 and i no longer have any icons on any button or menu !
Chronograf is deployed on CentOS VM and the result is the same on chrome, firefox and edge.
I have already cleared my navigator cash.

Any idea on the issue ?

Best regards

**Comment by gribnut** [comment]
I have found similar behavior with v1.7.1.   However, issue is triggered from reverse proxy config and dependency upon "BASE_PATH" environment variable set.  Appears that path to icons (and handful of other files referenced in css) are not using the "BASE_PATH" value which breaks icons when used in scenario that reverse proxy depends upon BASE_PATH as root URI.

**Comment by youmil** [comment]
Thank's for your contribution.
Do you fiond any solution or work around ?

**Comment by nhaugo** [comment]
We are working on a fix

**Comment by nhaugo** [comment]
Hi @youmil we are unable to reproduce this. Can you go into the debug console and do perform an empty cache and hard reload, then provide a screenshot of what chronograf looks like and also a screen shot of the network tab in the debug console.  Thanks for your help

**Comment by youmil** [comment]
Hi @nhaugo I have performed the empty cache and hard reload and the result is the  same.
Please note that chronograf is triggered through a reverse proxy.
Please find in attachment the screenshot of the network tab and the look and feel of chronograf.

![capture_network](https://user-images.githubusercontent.com/33926859/48251002-80fd0a80-e400-11e8-8775-1975d82648f8.JPG)
![capture_chronograph](https://user-images.githubusercontent.com/33926859/48251003-80fd0a80-e400-11e8-92bc-6caad80c4b35.JPG)

**Comment by jeanmatthieud** [comment]
Also seen with 1.7.1. Was working fine before.
I'm using "BASEBATH=/chronograf" in "/etc/default/chronograf".
Same result as [youmil](https://github.com/influxdata/chronograf/issues/4782#issuecomment-437284292)

**Comment by hctpbl** [comment]
Same problem here, using Nginx as a reverse proxy. Nginx configuration is the following:

```
    location ^~ /chronograf/ {
        proxy_ignore_client_abort on;
        proxy_pass http://127.0.0.1:8888;
        proxy_set_header Host $host;
    }
```

And I have set base path in `/etc/default/chronograf` to the fowllowing:
```
BASE_PATH=/chronograf
```

This is the result:
![captura de pantalla 2018-11-10 a las 12 37 41](https://user-images.githubusercontent.com/651015/48300899-77999e00-e4e5-11e8-888d-733f9316a68c.png)

Icon and font resources are being requested to urls like
```
https://<domain_name>/chronografRoboto-Regular.f82a1950.ttf
```

So a `/` is missing after the base path, this url works:

```
https://<domain_name>/chronograf/Roboto-Regular.f82a1950.ttf
```

**Comment by markspcs** [comment]
Same issue here. Using reverse proxy as well with BASEPATH=/chronograf . I can only use reverse proxy in my environment, so I had to rollback to 1.6  . Looking forward to 1.7 to fix my host vs hostname issue. So please let me know if you need anything additional to troubleshoot this for the next load.

**Comment by RenEvo** [comment]
BASE_PATH is definitely broken by not having the prefix path on the front end for fonts.

Verified across all of our 10x instances of chronograf with varying BASE_PATH configurations, trying to add the trailing slash `/debug/monitoring/` results in chronograf failing to start due to trailing slash on `BASE_PATH` configuration.

**Comment by gribnut** [comment]
Confirmed that BASE_PATH now functioning correctly since updating my instance to 1.7.3.

**Comment by nhaugo** [comment]
@gribnut thanks for letting us know

**Comment by russorat** [comment]
released in 1.7.3
